### DISCUSSION TITLE
How do I initialize Scriptkit on Windows startup?

### DISCUSSION THREAD

**Discussion by AmodeusR** [johnlindquist/kit#1134] [Q&A]
I didn't find an option for that in the settings. Are there anyway possible of doing this?

**Comment by Ambushfall** [comment]
Just like with any other program on windows, add it to startup based on your relevant OS Version.

Note: There currently isn't any app-native way to do this with a click of a button.

Manual mode is the way to go.

**Reply by AmodeusR** [reply]
I tried that already and it complains lack of files. Even if I copy them to the startup folder as well it keeps complaining about another file and so on. I came here asking this because I already tried the "manual way".

**Reply by johnlindquist** [reply]
@AmodeusR 

Kit.exe watches the  `~/.kit/db/app.json` file. When you change the "openAtLogin", it will trigger the `setLoginItemSettings()` (https://www.electronjs.org/docs/latest/api/app#appsetloginitemsettingssettings-macos-windows) with true/false.

The `~/.kit/logs/kit.log` should show a line like

```
Enable: Open Kit.app at login
```

> Note: This used to be on by default, then I refactored the settings and I believe the setting was cached in my system so I didn't realize it wasn't on by default anymore

**Reply by AmodeusR** [reply]
Though openAtLogin is set to true, the string you mentioned is nowhere to be found in the kit.log file. What does it mean?

**Reply by johnlindquist** [reply]
@AmodeusR Sorry, I meant to say, change it to `false`, save, then change it back to `true`, then save. It should appear in the logs then.

I'm fixing it for the next build so it triggers at app launch and file change instead of only file change.

**Reply by AmodeusR** [reply]
Oh, it showed up this time. So now it will initialize with windows, right? Thanks :)
